### PR TITLE
Use latest cpi dataset automatically

### PR DESCRIPTION
**Pull request by delenamalan**

### CODE REVIEW

**Comment by PiDelport** [comment]
Card: https://trello.com/c/CPKQZ3Px/64-use-latest-cpi-dataset-automatically

**Comment by jbothma** [comment]
No changes to data when building against this in staging.

File: budgetportal/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+from django.conf import settings
```
Comment (jbothma) [inline budgetportal/tests/test_models.py]: this suite should be called test_datasets now, rather than test_models

**Comment by delenamalan** [comment]
@jbothma Sounds right :+1:

**Pull request merged**